lib/rules/function-no-unknown/index.js

```diff
@@ -23,14 +25,34 @@ const meta = {
 };
 
 /** @type {import('stylelint').Rule} */
-const rule = (primary) => {
+const rule = (primary, secondaryOptions) => {
 	return (root, result) => {
-		const validOptions = validateOptions(result, ruleName, { actual: primary });
+		const validOptions = validateOptions(
+			result,
+			ruleName,
+			{ actual: primary },
+			{
+				actual: secondaryOptions,
+				possible: {
+					ignoreFunctions: [isString, isRegExp],
+				},
+				optional: true,
+			},
+		);
 
 		if (!validOptions) {
 			return;
 		}
 
+		/** @type {Array<string | RegExp>} */
+		const ignoreFunctions = (secondaryOptions && secondaryOptions.ignoreFunctions) || [];
+
+		/** @type {(funcName: string) => boolean} */
+		const isFunctionIgnored =
+			ignoreFunctions.length === 0
+				? () => false
+				: (funcName) => Boolean(matchesStringOrRegExp(funcName, ignoreFunctions));
+
```

Comment:
If possible, let's use the [`optionsMatches`](https://github.com/stylelint/stylelint/blob/main/lib/utils/optionsMatches.js) util here.

Reply:
Thanks! I forgot the util. 😅